planning/freespace_planning_algorithms/CMakeLists.txt

```diff
@@ -34,7 +34,13 @@ if(BUILD_TESTING)
   target_link_libraries(freespace_planning_algorithms-test
     freespace_planning_algorithms
   )
-
+  find_package(ament_cmake_ros REQUIRED)
+  ament_add_ros_isolated_gtest(freespace_planning_algorithms_astar-test
+    test/src/test_freespace_planning_algorithms_astar.cpp
```

Comment:
This file can just be added to the previous call of `ament_add_ros_isolated_gtest`.